js_functions/delete.js

```diff
@@ -0,0 +1,11 @@
+export default function deleteData(path) {
+  const options = {
+    method: "DELETE",
+  };
+  fetch(`https://boolean-api-server.fly.dev/Chloe0701962/${path}`, options)
```

Comment:
You reuse this string a few times. I'd probably put `https://boolean-api-server.fly.dev/Chloe0701962/` into a const named BASE_API_URL that you can import where needed